assets/scss/components/_schema-renderer.scss

```
@@ -0,0 +1,406 @@
+
+$schema-type-colors: (
+  string:  (#dbeafe, #1e40af, #1e3a5f, #93c5fd),
+  number:  (#fce7f3, #9f1239, #4a1942, #f9a8d4),
+  boolean: (#fef3c7, #92400e, #4a3728, #fde68a),
+  object:  (#e0e7ff, #3730a3, #2e2b5f, #c7d2fe),
+  array:   (#d1fae5, #065f46, #1a3a2a, #6ee7b7),
+);
+
+@each $type, $colors in $schema-type-colors {
+  .sr-type--#{$type} {
+    background-color: nth($colors, 1);
```

Stylelint (17.4.0)

[error] Expected list.nth instead of nth (scss/no-global-function-names)

(scss/no-global-function-names)